botocore/regions.py

```diff
@@ -46,6 +48,12 @@
 LOG = logging.getLogger(__name__)
 DEFAULT_URI_TEMPLATE = '{service}.{region}.{dnsSuffix}'  # noqa
 DEFAULT_SERVICE_DATA = {'endpoints': {}}
+# Allowed values for the ``account_id_endpoint_mode`` config field.
+VALID_ACCOUNT_ID_ENDPOINT_MODES = (
+    'preferred',
+    'disabled',
+    'required',
+)
```

Comment:
Do these actually need to be a public global constant? It looks like we have these relegated to the `EndpointRulesetResolver`.